libs/email/tsconfig.lib.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "./tsconfig.json",
   "include": ["src/**/*.ts"],
+  "compilerOptions": {
+    "jsx": "react"
```

Comment:
**blocker:** Should be `react-jsx`

Reply:
why?

Reply:
`react` was used in <17, `react-jsx` is newer and it should be used from React 17+.